docker-compose.yml

```diff
@@ -3,28 +3,32 @@ services:
         build:
             context: .
         volumes:
-            - .:/var/www/html
+            - '.:/var/www/html'
         depends_on:
             - mysql
         ports:
-          - 80:8080
+            - '80:8080'
     mysql:
-        image: mysql/mysql-server:8.0
+        image: 'mysql/mysql-server:8.0'
         ports:
-            - 3306:3306
+            - '3306:3306'
         environment:
             MYSQL_ROOT_PASSWORD: '${DB_PASSWORD}'
-            MYSQL_ROOT_HOST: "%"
             MYSQL_DATABASE: '${DB_DATABASE}'
             MYSQL_USER: '${DB_USERNAME}'
             MYSQL_PASSWORD: '${DB_PASSWORD}'
-            MYSQL_ALLOW_EMPTY_PASSWORD: 1
         volumes:
             - 'sailmysql:/var/lib/mysql'
         healthcheck:
-            test: ["CMD", "mysqladmin", "ping", "-p${DB_PASSWORD}"]
+            test:
+                - CMD
+                - mysqladmin
+                - ping
+                - '-p${DB_PASSWORD}'
             retries: 3
             timeout: 5s
 volumes:
     sailmysql:
         driver: local
+    sail-mysql:
+        driver: local
```

Comment:
Do we need this twice?

Reply:
No. Must've been generated automatically.